internal/cli/root.go

```diff
@@ -192,7 +192,11 @@ func (c *Client) Format() outputPkg.Format {
 	}
 	return format
 }
-
+func getMetalVersion() string {
```

Comment:
You'll want to use an IDE that does go formatting on save, or otherwise run this yourself before committing.

`gofmt -w -s .`

(I'm surprised the CI linters called out as much as it did but didn't catch this.)